cmd/spire-server/cli/bundle/delete.go

```diff
@@ -0,0 +1,55 @@
+package bundle
+
+import (
+	"context"
+	"errors"
+	"flag"
+
+	"github.com/mitchellh/cli"
+	"github.com/spiffe/spire/pkg/common/idutil"
+	"github.com/spiffe/spire/proto/api/registration"
+)
+
+// NewDeleteCommand creates a new "delete" subcommand for "bundle" command.
+func NewDeleteCommand() cli.Command {
+	return newDeleteCommand(defaultEnv, newClients)
+}
+
+func newDeleteCommand(env *env, clientsMaker clientsMaker) cli.Command {
+	return adaptCommand(env, clientsMaker, new(deleteCommand))
+}
+
+type deleteCommand struct {
+	// SPIFFE ID of the trust domain bundle
+	id string
+}
+
+func (c *deleteCommand) name() string {
+	return "bundle delete"
+}
+
+func (c *deleteCommand) synopsis() string {
+	return "Deletes bundle data"
+}
+
+func (c *deleteCommand) appendFlags(fs *flag.FlagSet) {
+	fs.StringVar(&c.id, "id", "", "SPIFFE ID of the trust domain")
+}
+
+func (c *deleteCommand) run(ctx context.Context, env *env, clients *clients) error {
+	if c.id == "" {
+		return errors.New("id is required")
```

Comment:
Maybe "SPIFFE ID of the trust domain is required" is more clear? It may not be obvious that is the SPIFFE ID and not other kind of id.